cdap-unit-test/src/main/resources/cdc.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
this shouldn't be in the cdap project

Reply:
Removed. Generating a basic config file for capabilities that do not have a config file.